### PR TITLE
* prompt to cl-user>

### PR DESCRIPTION
**Pull request by otjura**
complete with new reindenting

### CODE REVIEW

File: dates_and_times.md [hunk 1]
```diff
@@ -122,48 +122,48 @@ To sum up, we have seen how to turn a universal time into a calendar time and vi
...
-	 (format t ";;;  ~f seconds of run time~%"
-		 (/ (- ,run2 ,run1) internal-time-units-per-second))
-	 ,result)))
+CL-USER> (defmacro timing (&body forms)
+	   (let ((real1 (gensym))
+		   (real2 (gensym))
+		   (run1 (gensym))
+		   (run2 (gensym))
+		   (result (gensym)))
+	   `(letCL-USER> ((,real1 (get-internal-real-time))
```
Comment (pnathan) [inline dates_and_times.md]: This is an incorrect substitution.

File: files.md [hunk 2]
```diff
@@ -172,15 +172,15 @@ first argument is `NIL`, PEEK-CHAR will just return the next character that's
 waiting on the stream:
 
 ~~~lisp
-  * (with-input-from-string (stream "I'm not amused")
```
Comment (pnathan) [inline files.md]: Did you mean to remove this?

File: files.md [hunk 3]
```diff
@@ -199,28 +198,30 @@ had been read by READ-CHAR:
...
+#\n
+#\n
+#\o
 ~~~
 
 If the first argument to PEEK-CHAR is a character, the function will skip all
 characters until that particular character is found:
 
 ~~~lisp
-  * (with-input-from-string (stream "I'm not amused")
```
Comment (pnathan) [inline files.md]: As above - I think the intent of the text here is to show a REPL?

File: functions.md [hunk 4]
```diff
@@ -111,17 +106,17 @@ This explains the error message we got above - `(adder 3)` is neither a symbol n
...
 ~~~lisp
 ;;; continued from above
-* (funcall (adder 3) 5)
+CL-USER> (funcall (adder 3) 5)
 8
-* (apply (adder 3) '(5))
+CL-USER> (apply (adder 3) '(5))
 8
-* (defparameter *my-fun* (adder 3))
+CL-USER> (defparameter *my-funCL-USER> (adder 3))
```
Comment (pnathan) [inline functions.md]: incorrect substitution! :-(

File: functions.md [hunk 5]
```diff
@@ -130,37 +125,37 @@ Note that in the last example the function object returned by `(adder 3)` is sto
...
+CL-USER> (funcall (doubler '+) 4)
 8
-* (defparameter *my-plus* '+)
+CL-USER> (defparameter *my-plus* '+)
 *MY-PLUS*
-* (funcall (doubler *my-plus*) 4)
+CL-USER> (funcall (doubler *my-plus*) 4)
 8
-* (defparameter *my-fun* (doubler '+))
+CL-USER> (defparameter *my-funCL-USER> (doubler '+))
```
Comment (pnathan) [inline functions.md]: incorrect substitution

File: hashes.md [hunk 6]
```diff
@@ -30,16 +30,16 @@ The function [`gethash`](http://www.lispworks.com/documentation/HyperSpec/Body/f
 If you want to add an element to a hash table, you can use `gethash`, the function to retrieve elements from the hash table, in conjunction with [`setf`](http://www.lispworks.com/documentation/HyperSpec/Body/m_setf_.htm).
 
 ~~~lisp
-* (defparameter *my-hash* (make-hash-table))
+CL-USER> (defparameter *my-hashCL-USER> (make-hash-table))
```
Comment (pnathan) [inline hashes.md]: incorrect substitition

File: hashes.md [hunk 7]
```diff
@@ -100,24 +98,24 @@ In this case you'll have to check the _second_ return value of `gethash` which w
 Use [`remhash`](http://www.lispworks.com/documentation/HyperSpec/Body/f_remhas.htm) to delete a hash entry. Both the key and its associated value will be removed from the hash table. `remhash` returns T if there was such an entry, `nil` otherwise.
 
 ~~~lisp
-* (defparameter *my-hash* (make-hash-table))
+CL-USER> (defparameter *my-hashCL-USER> (make-hash-table))
```
Comment (pnathan) [inline hashes.md]: incorrect substitution.

**Comment by pnathan** [comment]
Some issues with the commits, but just quick-fix stuff. :-)

**Comment by otjura** [comment]
fixed

**Comment by pnathan** [comment]
Sorry about the delay! Last week got et by work. @otjura  - is this PR ready to go in? Please confirm, and I'll merge.

**Comment by otjura** [comment]
Yes, it should be.

**Pull request merged**